modules/ROOT/pages/runtime-fabric-logs.adoc

```diff
@@ -22,83 +22,264 @@ Runtime Fabric enables you to specify the level of severity of the message writt
+| Compress | Specifies compression of your UDP packets if the transport protocol is udp. | | true
+|===
+
+Example Fluent Bit output plugin configuration:
+```
+[OUTPUT]
+    Name                    gelf
+    Match                   kube.*
+    Host                    <your-graylog-server>
+    Port                    12201
+    Mode                    tcp
+    Gelf_Short_Message_Key  data
+```
+
+==== Splunk Configuration Parameters
+
+[%header%autowidth.spread,cols="a,a,a,a"]
+.Splunk Configuration Parameters
+|===
+| Key | Description | Required | Default value
+| Host | Specifies the IP address or hostname of the target Splunk service. | | 127.0.0.1
+| Port | Specifies the TCP port of the target Splunk service. | | 8088
+| Splunk_Token | Specifies the Authentication Token for the HTTP Event Collector interface. | |
+| Splunk_Send_Raw | When enabled, specifies the record keys and values to be set in the top level of the map instead of under the `event` key. | | Off
+| HTTP_User | Specifies an optional username for Basic Authentication on HEC. | |
+| HTTP_Passwd | Specifies a password for the user defined in `HTTP_User`. | |
+|===
+
+Example Fluent Bit output plugin configuration:
+```
+[OUTPUT]
+    Name        splunk
+    Match       *
+    Host        127.0.0.1
+    Port        8088
+    TLS         On
+    TLS.Verify  Off
+    Message_Key my_key
+```
+
+== Disable Log Forwarding for Specific Pods
+
+By default, Fluent Bit log forwarding is enabled on all pods (MuleApplication pods, Runtime Fabric pods and Infrastructure pods). 
+
+To disable log forwarding for specific pods, you can annotate it with `fluentbit.io/exclude`, as shown in the following example:
+```
+apiVersion: v1
+kind: Pod
+metadata:
+  name: apache-logs
+  labels:
+    app: apache-logs
+  annotations:
+    fluentbit.io/exclude: "true"
+spec:
+  containers:
+  - name: apache
+  ...
+```
+OR, you can use the following kubectl command to annotate it on pods in a specific namespace:
+```
+kubectl annotate pod <pod-name> -n <namespace> fluentbit.io/exclude="true"
+```
+Note that the annotation value is boolean and must be in quotation marks. 
+
+== Disable Anypoint Monitoring or Fluent Bit Log Forwarding:
+
+. Navigate to Runtime Manager and select *Applications*.
+. Select the application for which you want to disable log forwarding.
+. Select the *Logs* tab.
+. Deselect the applicable log forwarding options:
++
+** *Forward logs to Anypoint Monitoring* 
+** *Forward logs to <third party service>*
+
+[WARNING]
+Disabling log forwarding options deletes all the applicable configuration information from Runtime Fabric.
+
+== Forward Logs to Other External Services
+
+Anypoint Runtime Fabric enables you to forward application and cluster logs to external logging solutions other than the third party solutions supported using Fluent Bit log forwarding output plugins. The log forwarder built into Runtime Fabric enables you to send log data to an `rsyslog` server over TCP or UDP.
```

Comment:
```suggestion
Anypoint Runtime Fabric also enables you to forward application and cluster logs to an `rsyslog` server over TCP or UDP so that you can use an external central location for viewing, retention, and receiving alerts. Logging solutions such as Splunk or Logstash provide methods to receive log data from `rsyslog` clients.
```